### PR TITLE
Add age reporting in instrument.BuildReporter

### PR DESCRIPTION
**Pull request by prateek**
/cc @robskillington @jeromefroe

### CODE REVIEW

File: instrument/build.go [hunk 1]
```diff
@@ -59,10 +65,11 @@ var (
...
 func LogBuildInfo() {
-	log.Printf("Go Runtime version: %s\n", goVersion)
-	log.Printf("Build Revision:     %s\n", Revision)
-	log.Printf("Build Branch:       %s\n", Branch)
-	log.Printf("Build Date:         %s\n", BuildDate)
+	log.Printf("Go Runtime version:  %s\n", goVersion)
+	log.Printf("Build Revision:      %s\n", Revision)
+	log.Printf("Build Branch:        %s\n", Branch)
+	log.Printf("Build Date:          %s\n", BuildDate)
+	log.Printf("Build DateUnixNanos: %s\n", BuildDateUnixNanos)
```
Comment (xichen2020) [inline instrument/build.go]: nit: use `%d` instead of `%s` for int64s.

File: instrument/build.go [hunk 2]
```diff
@@ -41,15 +41,21 @@ var (
 	// BuildDate is the date this build was created.
 	BuildDate = "unknown"
 
+	// BuildDateUnixNanos is the UnixNanos since epoch representing the date this build was created.
+	BuildDateUnixNanos int64 = 0
```
Comment (xichen2020) [inline instrument/build.go]: nit: `BuildTimeUnixNanos` might be more apt. Also this should probably be a const?

File: instrument/build.go (hunk 1 above)
Comment (prateek) [reply instrument/build.go]: sure thing

File: instrument/build.go (hunk 2 above)
Comment (prateek) [reply instrument/build.go]: sure thing, will rename.

Re `const`: nah, needs to be a `var` to allow it to be overriden from ldflags at build time.

Comment (xichen2020) [reply instrument/build.go]: Ah yeah that's fair, might need to get rid of the 0 initial to silence lint.

File: instrument/build.go (hunk 1 above)
Comment (prateek) [reply instrument/build.go]: fixed

File: instrument/build.go (hunk 2 above)
Comment (prateek) [reply instrument/build.go]: yep sure thing. fixed both issues.

**Pull request merged**